src/algebra/covariant_and_contravariant.lean

```diff
@@ -203,32 +203,32 @@ variables {α : Type*} {M N μ} [preorder α] [preorder N]
 variable {f : N → α}
 
 /-- The partial application of a constant to a covariant operator is monotone. -/
-lemma covariant.monotone_of_const [covariant_class M N μ (≤)] (m : M) : monotone (μ m) :=
+lemma covariant.monotone_of_const (μ) [covariant_class M N μ (≤)] {m : M} : monotone (μ m) :=
```

Comment:
If you want to make a change here, the best solution is probably to make both mu and m explicit, as neither of them can be guessed by Lean.

Reply:
For context, I changed this to avoid needing to use `@` for the following line in the proof of `monotone.covariant_of_const_left_nat`:

```lean
obtain h | ⟨N, hN⟩ := monotone.tendsto_at_top_at_top_or_eventually_const_int
    (covariant.monotone_of_const μ),
```

It does seem to be able to infer `m` here. Do you still want me to make both explicit?